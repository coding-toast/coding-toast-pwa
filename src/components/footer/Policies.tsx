import * as React from 'react';
import { Row, Col, Container } from 'react-bootstrap';
import { PrivacyPolicy, TermsOfService } from 'components';

const Policies: React.FC = () => {
  return (
    <Container style={{ maxWidth: '30em' }}>
      <Row>
        <Col className='text-left'>
          <PrivacyPolicy muted />
        </Col>
        <Col className='text-right'>
          <TermsOfService muted />
        </Col>
      </Row>
    </Container>
  );
};

export default Policies;
