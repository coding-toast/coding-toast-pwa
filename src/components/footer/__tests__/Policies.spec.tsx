import * as React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Policies from '../Policies';

describe('<Policies />', () => {
  it('renders correctly', () => {
    const component = shallow(<Policies />);

    expect(toJson(component)).toMatchSnapshot();
  });
});
