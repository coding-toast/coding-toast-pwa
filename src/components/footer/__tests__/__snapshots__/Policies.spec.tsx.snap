// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Policies /> renders correctly 1`] = `
<Container
  fluid={false}
  style={
    Object {
      "maxWidth": "30em",
    }
  }
>
  <Row
    noGutters={false}
  >
    <Col
      className="text-left"
    >
      <mockConstructor
        className="text-muted"
        to="/"
      >
        Privacy Policy
      </mockConstructor>
    </Col>
    <Col
      className="text-right"
    >
      <mockConstructor
        className="text-muted"
        to="/"
      >
        Terms of Service
      </mockConstructor>
    </Col>
  </Row>
</Container>
`;
